error.go

```diff
@@ -221,3 +221,29 @@ var tooManyResultsError = &microerror.Error{
 func IsTooManyResults(err error) bool {
 	return microerror.Cause(err) == tooManyResultsError
 }
+
+var (
+	yamlConvertingFailedPrefix = "error converting YAML to JSON:"
```

Comment:
```suggestion
	yamlConversionFailedErrorText = "error converting YAML to JSON:"
```

As I said in my previous review I prefer `Conversion` over `Converting`.  As it matches our usual naming conventions.

Also you're using `strings.Contains` not `strings.HasPrefix`. So I think having prefix in the name is confusing.

Reply:
I doubt that I could use `HasPrefix` since the keywords would be located in the middle of the sentence. 
```rpc error: code = Unknown desc = YAML parse error on kubernetes-nginx-ingress-controller-chart/templates/configmap.yaml: error converting YAML to JSON: yaml: line 16: did not find expected key```
Maybe we should change the name of `prefix` to something else?

Reply:
Exactly this isn't a prefix because its not at the start of the error message. That's why I suggested using `ErrorText` in the name. 

https://dictionary.cambridge.org/grammar/british-grammar/word-formation/prefixes